src/EnergyPlus/SolarShading.cc

```diff
@@ -3691,12 +3691,12 @@ namespace SolarShading {
 			WithShdgHoriz = 0.;
 			WoShdgHoriz = 0.;
 
-			for ( int IPhi = 1; IPhi <= NPhi; ++IPhi ) { // Loop over patch altitude values
-				SUNCOS( 3 ) = SinPhi[ IPhi ];
+			for ( int IPhi = 0; IPhi < NPhi; ++IPhi ) { // Loop over patch altitude values
+				SUNCOS( 3 ) = SinPhi[IPhi];
```

Comment:
Probably should add them, but I'd guess that @Myoldmopar has the final say. As with everything, there's a variety with and without throughout the code.